src/serializers/locations-serializer.js

```diff
@@ -7,18 +7,38 @@ import { paginate } from 'utils/paginator';
 import { apiBaseUrl, resourcePathLink, paramsLink } from 'utils/uri-builder';
 
 const locationResourceProp = openapi.components.schemas.LocationResource.properties;
+const serviceResourceProp = openapi.components.schemas.ServiceResource.properties;
 const locationResourceType = locationResourceProp.type.enum[0];
 const locationResourceKeys = _.keys(locationResourceProp.attributes.properties);
+const serviceResourceKeys = _.keys(serviceResourceProp.attributes.properties);
 const locationResourcePath = 'locations';
```

Comment:
Just my opinion, I feel like we should group these vars by resources e.g. https://github.com/osu-mist/onbase-api/blob/master/src/api/v1/serializers/onbase-serializer.js#L8-L21